modules/uhttp/http_options.go

```diff
@@ -20,17 +20,17 @@
 
 package uhttp
 
-// ModuleOption is a function that configures module creation.
-type ModuleOption func(*moduleOptions) error
+// ModuleOptionFn is a function that configures module creation.
+type ModuleOptionFn func(*moduleOption) error
```

Comment:
The user does not use this directly. I prefer Fn for aliases that represent the function. We can add an interface and name it ModuleOption and expose that if you prefer that.